interface/orders/receive_hl7_results.inc.php

```diff
@@ -751,6 +751,8 @@ function receive_hl7_results(&$hl7, &$matchreq, $lab_id = 0, $direction = 'B', $
             $mdm_docname = rhl7Text($a[12]);
         } else if ($a[0] == 'NTE' && ($context == 'ORC' || $context == 'TXA')) {
             // Is this ever used?
+        } else if ($a[0] == 'NTE' && ($context == 'PID')) {
```

Comment:
This segment is one of those that different labs handle in different ways. Labcorps I believe will send nte in pid segment and I assume others as well because it's current placement wouldn't be as it is.
I've certified with several labs and had to deal with this so i'm going to send my script and maybe you can glean from how I handled.
Perhaps will stop coming back to this for a different lab later.

Reply:
Jerry I think Stephen is just ignoring this case, to avoid the error message about a "misplaced or unknown" segment name.

Reply:
yep, I get that and moving out of pid segment meets spec but i've found labs don't follow spec and will send nte in the pid segment.
The solution imo is to give nte a context conditional ie 
`else if ($a[0] == 'NTE' && 'PID' == $context) {
            // will get orderid on save.
            $amain[0]['rep']['report_notes'] .= rhl7Text($a[3], true) . "\n";
        }`
For when nte is sent with pid.

Reply:
also can't forget obx vs nte.

Reply:
Good idea, not losing the note. :)

Reply:
By the way Rod when I wrote my Quest module I used the Electronic Reports pretty much as is (fix orphaned orders some etc). The comment I got from their cert team is this was the best presentation they've seen so, Kudo's my friend.

Reply:
That's quite a compliment, thanks!